tests/test_pyimager.py

```diff
@@ -1 +1,15 @@
-from pyimager import pyimager
+from pyimager.pyimager import *
+
+def test_stype():
+    img_array = reducolor(0, 'tests/mandrill.jpg')
+    color_n = len(np.unique(img_array.reshape(np.prod(img_array.shape[:2]),3), axis=0))
+    assert color_n == 2, f'Should return two colors only, {color_n} colors are returned'
+    img_array = reducolor(1, 'tests/mandrill.jpg')
+    color_n = len(np.unique(img_array.reshape(np.prod(img_array.shape[:2]),3), axis=0))
+    assert color_n == 24, f'Should return eight colors, {color_n} colors are returned'    
```

Comment:
seems to be 8 as your message described 

```suggestion
    assert color_n == 8, f'Should return eight colors, {color_n} colors are returned'    
```